src/Uno.UWP/Helpers/AppDataUriEvaluator.cs

```diff
@@ -16,6 +16,12 @@ internal static class AppDataUriEvaluator
 		private const string RoamingFolderRoute = "roaming";
 		private const string TemporaryFolderRoute = "temp";
 
+		/// <summary>
+		/// Determines if an URI is using ms-appdata
+		/// </summary>
+		public static bool IsAppDataUri(Uri uri)
+			=> uri.Scheme.Equals("ms-appdata", StringComparison.OrdinalIgnoreCase);
+
```

Comment:
This is already implemented: https://github.com/unoplatform/uno/blob/69095eb1b461251a61f470ab93ceac2c34f87eac/src/Uno.Foundation/Extensions/UriExtensions.cs#L50-L58